redbot/cogs/mod/kickban.py

```diff
@@ -430,7 +430,7 @@ async def tempban(
         self,
         ctx: commands.Context,
         user: discord.Member,
-        duration: UserInputOptional[commands.TimedeltaConverter] = timedelta(days=1),
+        duration: UserInputOptional[commands.TimedeltaConverter] = None,
```

Comment:
Type hint should look like this with the new default:
```suggestion
        duration: Optional[commands.TimedeltaConverter] = None,
```